src/Collection.php

```diff
@@ -936,6 +937,31 @@ public function updateOne($filter, $update, array $options = [])
         return $operation->execute($server);
     }
 
+    /*
+     * ChangeStream outline
+     *
+     * @see ChangeStreamCommand::__construct() for supported options
+     * @param array          $pipeline       List of pipeline operations
+     * @param array          $options        Command options
+     * @throws InvalidArgumentException for parameter/option parsing errors
```

Comment:
Missing a `@return` annotation here. I might also suggest `use MongoDB\ChangeStream as ChangeStreamResult` up top and then using `@return ChangeStreamResult` here, as in the Operation class.